### ISSUE TITLE
Argument "args" to "send_task" of "Celery" has incompatible type "Tuple[str, str, str]"; expected "Optional[Tuple[Any]]"

### ISSUE THREAD

**Issue by mdantonio** [sbdchd/celery-types#16]
Hello,
i have tasks with multiple args but it seems that the type hints only accepts a single argument

```
Argument "args" to "send_task" of "Celery" has incompatible type "Tuple[str, str, str]"; expected "Optional[Tuple[Any]]"
```

the provided type annotation is:

```
    def send_task(
        self,
        name: str,
        args: Optional[Tuple[Any]] = ...,
[...]
```

maybe that it should be

```
        args: Optional[Tuple[Any, ...]] = ...,
```

to allow for args of any length?

**Comment by sbdchd** [comment]
Thanks for the report!

Should be fixed in version 0.5.0 on pypi

**Comment by mdantonio** [comment]
Thank you, I can confirm the fix!
